mappings/none/bei.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS none/bei
+	FIELD a MAP_ICONS Lnone/kp;
```

Comment:
If this is a texture it should have `_TEX` suffix.
(Though maybe we should use a different term for texture maps that contain multiple sprites/icons.)